src/platform/unittest/TlsTest.cpp

```diff
@@ -19,6 +22,8 @@ const uint32_t DefaultFragmentSize = 1200;
 
 const uint8_t Alpn[] = { 1, 'A' };
 const uint8_t MultiAlpn[] = { 1, 'C', 1, 'A', 1, 'B' };
+const char* TestPath = PfxTestPath;
```

Comment:
I prefer to use all caps for C #defined variables.

Reply:
updated. I also added comment with explanation.